.github/ISSUE_TEMPLATE/navigator_feature_request.yml

```diff
@@ -0,0 +1,61 @@
+name: Navigator Feature Request
+description: Create a feature request to make Navigator even better
+labels: ['type-feature', 'pkg-navigator']
```

Comment:
Same sync with Jira problem described in comment for the bug report.

Also, we already have some other types like type-support and type-enhancement that could be used for this form for more specific issue classification. But this will require manual labeling too.

As for me, I'd rather leave the issue without a type label until someone reviews it.

Reply:
This is specifically for the feature request issue though, so we can assume that it will be that more often than not so I think we should keep that as a default. 

We maybe should add a `Navigator miscellaneous` issue?

There are two labels that might be redundant: `type-feature` and `type-enhancement`. I don't think we need both? There are only 3 with the `type-feature` tag

Reply:
Hard to say from the previous usage of labels since that there was no strict process. For now, we can leave it as is with forms for type-bug and type-feature. If we'll see a lot of issues that don't fall into these two groups we may create another one.